modules/oe_paragraphs_illustrations_lists/tests/modules/oe_paragraphs_illustrations_lists_test/oe_paragraphs_illustrations_lists_test.services.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  oe_paragraphs_illustrations_lists_test.column_options_event_subscriber:
+    class: Drupal\oe_paragraphs_illustrations_lists_test\EventSubscriber\ColumnOptionsEventTestSubscriber
+    tags:
+      - { name: 'event_subscriber' }
+  oe_paragraphs_illustrations_lists_test.ratio_options_event_subscriber:
+    class: Drupal\oe_paragraphs_illustrations_lists_test\EventSubscriber\RatioOptionsEventTestSubscriber
+    tags:
+      - { name: 'event_subscriber' }
```

Comment:
Don't see the reason why we can't have a single event subscriber for providing options

Reply:
Good point